utils/flakiness-dashboard/processing/dashboard_compressed_v1.js

```diff
@@ -64,7 +64,7 @@ function compressReports(reports) {
       let specObject = specs.get(specId);
       if (!specObject) {
         specObject = {
-          title: spec.title,
+          title: spec.titlePath.join(' › '),
```

Comment:
IIRC the `titlePath` yields a pretty long unreadable string, so I'd rather fix our tests to have nice titles instead.